integration/test/chart/basic/basic_test.go

```diff
@@ -0,0 +1,145 @@
+// +build k8srequired
+
+package basic
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/giantswarm/e2e-harness/pkg/release"
+	"github.com/giantswarm/e2etemplates/pkg/chartvalues"
+
+	"github.com/giantswarm/chart-operator/integration/key"
+)
+
+// TestChartLifecycle tests a Helm release can be created, updated and deleted
+// uaing a chart CR processed by chart-operator.
+//
+// - Create chart CR using apiextensions-chart-e2e-chart.
+// - Ensure test app specfied in the chart CR is deployed.
+//
+// - Update chart CR using apiextensions-chart-e2e-chart.
+// - Ensure test app is redeployed using updated chart tarball.
+//
+// - Delete apiextensions-chart-e2e-chart.
+// - Ensure test app is deleted.
+//
+func TestChartLifecycle(t *testing.T) {
+	ctx := context.Background()
+
+	// Test creation.
+	{
+		config.Logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chart %#q", key.CustomResourceReleaseName()))
+
+		c := chartvalues.APIExtensionsChartE2EConfig{
+			Chart: chartvalues.APIExtensionsChartE2EConfigChart{
+				Name:       key.TestAppReleaseName(),
+				Namespace:  "giantswarm",
+				TarballURL: "https://giantswarm.github.com/sample-catalog/kubernetes-test-app-chart-0.5.3.tgz",
+			},
+			ChartOperator: chartvalues.APIExtensionsChartE2EConfigChartOperator{
+				Version: "1.0.0",
+			},
+			Namespace: "giantswarm",
+		}
+
+		chartValues, err := chartvalues.NewAPIExtensionsChartE2E(c)
+		if err != nil {
+			t.Fatalf("expected %#v got %#v", nil, err)
+		}
+
+		chartInfo := release.NewStableChartInfo(key.CustomResourceReleaseName())
+		err = config.Release.Install(ctx, key.CustomResourceReleaseName(), chartInfo, chartValues)
+		if err != nil {
+			t.Fatalf("expected %#v got %#v", nil, err)
+		}
+
+		err = config.Release.WaitForStatus(ctx, fmt.Sprintf("%s-%s", "giantswarm", key.CustomResourceReleaseName()), "DEPLOYED")
```

Comment:
This is a bit of a pain. The release package prefixes the release name with the namespace. Except for the wait methods so this is needed.